packages/nc-gui-v2/plugins/init.ts

```diff
@@ -0,0 +1,46 @@
+import type { Dayjs } from 'dayjs'
+import dayjs from 'dayjs'
+import duration from 'dayjs/plugin/duration'
+import { defineNuxtPlugin } from '#app'
+
+dayjs.extend(duration)
+
+const handleFeedbackForm = async () => {
+  let { feedbackForm: currentFeedbackForm } = $(useGlobalState())
```

Comment:
Again: Do *not* call `useGlobalState` (as indicated by the comments in the composable itself) but extract the `$state` from `useNuxtApp`